CodeHS/6/3/8/Area-of-a-square-with-default-parameters-Travis.py

```diff
@@ -0,0 +1,10 @@
+def calculate_area(side_length = 10):
+    print side_length
+    print side_length * side_length
+
+user_length = int(input("Enter side length: "))
+
+if user_length <= 0:
+    calculate_area()
+else:
+    calculate_area(user_length)
```

Comment:
You may want to add a propper EOF here but this otherwise looks good

Reply:
What would be an EOF in this? 

also the codehs compiler is having a hard time with a few things even though my code is logically correct
![image](https://user-images.githubusercontent.com/35313221/69437062-8c238f80-0d10-11ea-948f-f5e980114590.png)

Reply:
CodeHS is dumb, and takes weird fiddling to make it work, your code is fine so id say dont stress yourself, just fiddle with extra text till it functions, codeHS does not expect elegant solutions
just press enter at the bottom of your file so you have a blank line